WMF Framework/SharedContainerCache.swift

```diff
@@ -81,3 +81,12 @@ public final class SharedContainerCache<T: Codable>: SharedContainerCacheHouseke
 @objc public protocol SharedContainerCacheHousekeepingProtocol: AnyObject {
     static func deleteStaleCachedItems(in subdirectoryPathComponent: String)
 }
+
+@objc public class SharedContainerCacheClearFeaturedArticleWrapper: NSObject {
+    @objc public static func clearOutFeaturedArticleWidgetCache() {
+        let sharedCache = SharedContainerCache<WidgetCache>(fileName: "Widget Cache", defaultCache: { WidgetCache(settings: .default, featuredContent: nil) })
```

Comment:
With our multiple uses of the "Widget Cache" filename, do you think we'd be better served making it a static property on `WidgetCache` (or leveraging `SharedContainerCacheCommonNames`, though I think the intent of the properties there are slightly different)?

Reply:
Ah good catch, I think that is exactly what `SharedContainerCacheCommonNames` was made for, if I recall correctly. This is done!